src/org/jitsi/impl/neomedia/transform/RtxTransformer.java

```diff
@@ -554,6 +554,70 @@ private void nackReceived(long mediaSSRC, Collection<Integer> lostPackets)
         }
     }
 
+    public void pad(long ssrc, long bytes)
+    {
+        StreamRTPManager receiveRTPManager = mediaStream
+            .getRTPTranslator()
+            .findStreamRTPManagerByReceiveSSRC((int) ssrc);
+
+        MediaStreamTrackReceiver receiver = null;
+        if (receiveRTPManager != null)
+        {
+            MediaStream receiveStream = receiveRTPManager.getMediaStream();
+            if (receiveStream != null)
+            {
+                receiver = receiveStream.getMediaStreamTrackReceiver();
+            }
+        }
+
+        if (receiver == null)
+        {
+            return;
+        }
+
+        RTPEncodingDesc encoding = receiver.findRTPEncodingDesc(ssrc);
+        if (encoding == null)
+        {
+            logger.warn("encoding_not_found"
+                + ",stream_hash=" + mediaStream.hashCode()
+                + " ssrc=" + ssrc);
+            return;
+        }
+
+        long rtxSSRC = encoding.getRTXSSRC();
+
+        int pktLen = RawPacket.FIXED_HEADER_SIZE + 0xFF;
+        // int mod = (int) (bytes % pktLen);
+        int len = (int) (bytes / pktLen) + 1;
+        RawPacket[] pkts = new RawPacket[len + 1];
```

Comment:
How about extracting a method to send a single padding packet with a given length? We could then easily handle the remaining bytes too